README.md

```diff
@@ -1,22 +1,52 @@
 # VirtoCommerce.Cart
-VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager. 
 
-# Documentation
-User guide:
+## Overview
 
-Developer guide:
+VirtoCommerce.Cart module represents shopping cart management system. This module doesn't have any UI in VC Manager.
 
-# Installation
+Shopping cart module manages customers accumulated list of items, calculates a total for the order, including shipping and handling (i.e., postage and packing) charges and the associated taxes.
+
+![Cart Module](docs/media/screen-cart-module.png)
+
+![Cart Module Info](docs/media/screen-cart-module-info.png)
+
+## Features
+
+1. Supports multiple carts
+1. Wishlist
+1. Grouping multiple carts to one order
+1. Anonymous carts
+1. Stock reservation
+1. Multiple payments methods
+1. Create new cart from orders history
+
+The main purpose of the Cart module is to implement customer shopping cart management in VC eCommerce solution. It encapsulates data persistence, management services and exposes REST API endpoints.
+
+The cart module is connected with the shopping cart API requests:
+
+1. General API request that allows saving the entire shopping cart information on the database;
+1. Specific API requests that allow call specific operations, for example delete, edit, choose payment method, choose delivery type, etc.
```

Comment:
@eMazeika @mayaalsamarrai For discuss. I think it would be nice to add a list of API functions and their brief description

Reply:
link to live swagger would be better, imho. Can add descriptions in api level, if missing.

Reply:
Guys, what's to technical stuff, like description in api level, etc, I would ask you to help me with that. Please provide the details and I will add it to the document

Reply:
To describe the API, you should add "Documentation comments" https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments
You can find examples in /VirtoCommerce.CartModule.Web/Controllers/Api/CartModuleController.cs
Its look like 

/// <summary>
/// Get shopping cart by id
/// </summary>

Reply:
A task in Jira created